Server Side Consistency Docs (#14392)
hashicorp/vault: website/content/docs/configuration/replication.mdx

- `best_effort_wal_wait_duration` `(duration: "2s")` - Legacy mitigation to try to prevent stale reads: when a
  write is made via RPC, i.e. the node handling the request didn't do the write itself, wait up to this much
  time to see the resulting WAL present locally before returning a response to the client.

> - `allow_forwarding_via_token` `(string: "")` - When set to `new_token`, causes requests sent to non-active nodes

Reviewer: ```suggestion
- `allow_forwarding_via_token` `(string: "")` - When set to `new_token`, requests are sent to non-active nodes
```

> - `allow_forwarding_via_token` `(string: "")` - When set to `new_token`, causes requests sent to non-active nodes

Reviewer: ```suggestion
- `allow_forwarding_via_token` `(string: "")` - When set to `new_token`,  requests sent to non-active nodes
```

>   to be forwarded if the node does not yet have the token information in storage. 

Reviewer: ```suggestion
and forwarded if the node does not yet have the token information in storage. 
```

>   to be forwarded if the node does not yet have the token information in storage. 

Reviewer: ```suggestion
  are forwarded if the node does not yet have the token information in storage. 
```
